src/commands/utils/eval.js

```diff
@@ -0,0 +1,43 @@
+const Command = require('../../classes/Command')
+
+module.exports = new Command((msg, suffix) => {
+  const util = require('util')
+  try {
+    const returned = eval(suffix) // eslint-disable-line no-eval
+    let str = util.inspect(returned, {
+      depth: 1
+    })
+    if (str.length > 1900) {
+      str = str.substr(0, 1897)
+      str = str + '...'
+    }
+    str = str.replace(new RegExp(process.env.BOT_TOKEN, 'gi'), '( ͡° ͜ʖ ͡°)')
+    msg.channel.createMessage('```js\n' + str + '\n```').then((ms) => {
+      if (returned !== undefined && returned !== null && typeof returned.then === 'function') {
+        returned.then(() => {
+          let str = util.inspect(returned, {
+            depth: 1
+          })
+          if (str.length > 1900) {
+            str = str.substr(0, 1897)
+            str = str + '...'
+          }
+          ms.edit('```js\n' + str + '\n```')
+        }, (e) => {
+          let str = util.inspect(e, {
+            depth: 1
+          })
+          if (str.length > 1900) {
+            str = str.substr(0, 1897)
+            str = str + '...'
+          }
+          ms.edit('```js\n' + str + '\n```')
+        })
+      }
```

Comment:
This block could use some documentation... Why are these lines doing the things they do and what do the values within them represent?